### PR TITLE
fix: 修复 书趣阁 路由失效 / fix shuquge route failure

### PR DESCRIPTION
**Pull request by ActonGen**
fix shuquge route failure

## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/biquge/https://www.ishuquge.com/txt/8659/
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [x] `Puppeteer`

## 说明 / Note
修复 书趣阁 路由失效
fix shuquge route failure

该路由必须启用 Puppeteer 其他俩种方式无效
The route must enable 'Puppeter' The other two methods are invalid

### CODE REVIEW

File: docs/reading.md [hunk 1]
```diff
@@ -511,8 +511,7 @@ count 的取值范围为 1-12，为防止请求次数过多，推荐设置为 5
 <Route author="ActonGen" example="/novel/shuquge/8659" path="/novel/shuquge/:id" :paramsDesc="['小说 id, 可在对应小说页 URL 中找到, 例如 `8659`']">
 
 举例网址：<http://www.shuquge.com/txt/8659/index.html>
-
-</Route>
+需启用 puppeteer 并使用国内网络环境 </Route>
```
Comment (TonyRL) [inline docs/reading.md:514]: Please add `puppeteer=1` to route tag

Comment (ActonGen) [reply docs/reading.md:514]: modified

**Review by TonyRL — COMMENTED** [review]
I don't see the necessity of using `puppeteer`. I'm able to use `got` to retrieve the feed and its content without any issue from Singapore IP

**Comment by ActonGen** [comment]
> I don't see the necessity of using `puppeteer`. I'm able to use `got` to retrieve the feed and its content without any issue from Singapore IP

I don't know what happened. Using got in my environment will cause timeout, but the browser can be opened normally. Based on this judgment, I used the puppeteer

image
image

timeout

image

**Comment by TonyRL** [comment]
<details>
<summary>Screenshot</summary>

image
</details>

**Comment by ActonGen** [comment]
> 

should i close this pr

**Review by TonyRL — COMMENTED** [review]
You can simply replace the domain name with `https://www.ishuquge.com/txt/`

**Review by TonyRL — COMMENTED** [review]
Removed the old route. Please use the generic route for similar sites instead.

**Pull request merged**